backend/spec_router.py

```
@@ -0,0 +1,71 @@
+
+log = logging.getLogger("spec_router")
+
+
+def build_spec_router(get_current_user: Callable[..., Any]) -> APIRouter:
+    router = APIRouter(prefix="/api/specs", tags=["specs"])
+
+    @router.get("")
+    async def list_spec_artifacts(
+        status: str | None = Query(default=None),
+        limit: int = Query(default=50, ge=1, le=200),
+        user: dict = Depends(get_current_user),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)